frontend/src/scenes/authentication/signup/signupForm/signupLogic.ts

```diff
@@ -83,14 +84,20 @@ export const signupLogic = kea<signupLogicType>([
                 role_at_organization: '',
                 referral_source: '',
             } as SignupForm,
-            errors: ({ first_name, organization_name }) => ({
+            errors: ({ first_name }) => ({
                 first_name: !first_name ? 'Please enter your name' : undefined,
-                organization_name: !organization_name ? 'Please enter your organization name' : undefined,
             }),
             submit: async (payload, breakpoint) => {
                 breakpoint()
                 try {
-                    const res = await api.create('api/signup/', { ...values.signupPanel1, ...payload })
+                    const res = await api.create('api/signup/', {
```

Comment:
Should we update the placdeholder for org to match the default: frontend/src/scenes/authentication/signup/signupForm/panels/SignupPanel2.tsx?

Reply:
Good call, yeah I can do that.